README.md

```diff
@@ -246,14 +246,15 @@ The following `create` parameters are specific to the `tsdb` backend and are pas
   - **Valid Values:** A string of the format `"[0-9]+/[smh]"` &mdash; where '`s`' = seconds, '`m`' = minutes, and '`h`' = hours.
     For example, `"1/s"` (one sample per minute), `"20/m"` (20 samples per minute), or `"50/h"` (50 samples per hour).
 
-- <a id="method-create-tsdb-param-aggregates"></a>**aggregates** &mdash; Default aggregates to calculate in real time during the samples ingestion.
+- <a id="method-create-tsdb-param-aggregates"></a>**aggregates** &mdash; A list of aggregation functions for executing in real time during the samples ingestion ("pre-aggregation").
 
   - **Type:** `str`
   - **Requirement:** Optional
   - **Valid Values:** A string containing a comma-separated list of supported aggregation functions &mdash; `avg`| `count`| `last`| `max`| `min`| `rate`| `stddev`| `stdvar`| `sum`.
     For example, `"count,avg,min,max"`.
+  - **Default Value:** `"1h"` (1 hour)
```

Comment:
it's the default value for the granularity, not aggregates.

Reply:
@talIguaz FIXED in commit dd62493.